common/djangoapps/student/tests/test_bulk_email_settings.py

```diff
@@ -51,34 +51,46 @@ def setUp(self):
             name=self.course.display_name.replace(' ', '_'),
         )
 
-    @patch.dict(settings.FEATURES, {'ENABLE_INSTRUCTOR_EMAIL': True, 'REQUIRE_COURSE_EMAIL_AUTH': False})
+    @patch(
+        'bulk_email.models.BulkEmailFlag.current',
+        Mock(return_value=BulkEmailFlag(enabled=True, require_course_email_auth=False))
+    )
```

Comment:
I moved the patching of the setting to the test class instead of the test method where I was able, but a good number of these tests are explicitly relying on having different values. I kept those patches at the test level.